engine/src/render/tone_mapping/plugin.cpp

```diff
@@ -74,11 +94,21 @@ void cubos::engine::toneMappingPlugin(Cubos& cubos)
     cubos.system("apply Tone Mapping to the HDR texture")
         .tagged(drawToRenderTargetTag)
         .tagged(toneMappingTag)
-        .call([](const State& state, const Window& window, Query<RenderTarget&, const HDR&, const ToneMapping&> query) {
+        .call([](const State& state, const Window& window,
+                 Query<RenderTarget&, const HDR&, const ToneMapping&, Opt<const FXAA&>> query) {
             auto& rd = window->renderDevice();
 
-            for (auto [target, hdr, toneMapping] : query)
+            for (auto [target, hdr, toneMapping, fxaa] : query)
             {
+                if (fxaa.contains())
+                {
+                    FxaaConfig fxaaConfig{};
+                    fxaaConfig.edgeThresholdMin = fxaa->edgeThresholdMin;
+                    fxaaConfig.edgeThresholdMax = fxaa->edgeThresholdMax;
+                    fxaaConfig.subpixelQuality = fxaa->subpixelQuality;
+                    fxaaConfig.iterations = fxaa->iterations;
+                    state.fxaaConfigCB->fill(&fxaaConfig, sizeof(FxaaConfig));
+                }
```

Comment:
I may be missing something, but this seems to leave the buffer completely uninitialized if the FXAA component isn't present, which in that case could lead to unexpected behavior.